changelog/unreleased/3569

```diff
@@ -0,0 +1,7 @@
+Enhancement: Improvements for the UI in the passcode screen
+
+Redesign of the passcode screen to have the numeric keyboard in the screen instead of using the Android one.
+Linear and Relatives layouts were deleted, now only ConstraintLayout is used.
```

Comment:
I think that this detailed information is not relevant to the user, think that there may be people who do not know anything about layout.